.github/workflows/coverage.yml

```diff
@@ -30,28 +30,15 @@ jobs:
       - name: Install rustfilt symbol demangler
         run: |
           cargo install rustfilt
-      - name: Rerun tests for coverage
+
+      - name: Install cargo-llvm-cov cargo command
+        run: |
+          cargo install cargo-llvm-cov --version 0.1.0-alpha.3
+
+      - name: Generate code coverage
         env:
-          RUSTFLAGS: -Zinstrument-coverage -C link-dead-code -C debuginfo=2
-          LLVM_PROFILE_FILE: "${{ github.workspace }}/test.%p.profraw"
           ZEBRA_SKIP_NETWORK_TESTS: 1
-          CARGO_INCREMENTAL: 0
```

Comment:
We disabled incremental builds to avoid running out of disk space.

In the meantime, nightly disabled incremental builds to avoid incremental errors and miscompiles. So we should keep disabling incremental for now - otherwise our builds might fail when nightly turns it back on.

Reply:
Done ✔️